thundra_demo_localstack/README.md

```diff
@@ -0,0 +1,100 @@
+<!--
```

Comment:
No need to keep this readme file as it is relevant

Reply:
@gokhan721 what about this one?

Reply:
This readme file is automatically generated by Serverless. I have removed it.